### PR TITLE
Add [MicrobadgerSize] documentation

### PR DESCRIPTION
**Pull request by PyvesB**
Adding some documentation to help users, as per #3988.

### CODE REVIEW

File: services/microbadger/microbadger-size.service.js [hunk 1]
```diff
@@ -4,6 +4,29 @@ const prettyBytes = require('pretty-bytes')
 const BaseMicrobadgerService = require('./microbadger-base')
 const { NotFound } = require('..')
 
+const documentation = `
+<p>
+  The MicroBadger API can sometimes be a bit temparemental when it comes to retrieving the size of your image.
```
Comment (calebcartwright) [inline services/microbadger/microbadger-size.service.js]:
```suggestion
  The MicroBadger API can sometimes be a bit temperamental when it comes to retrieving the size of your image. 
```

Comment (PyvesB) [reply services/microbadger/microbadger-size.service.js]: Good spot!

File: services/microbadger/microbadger-size.service.js [hunk 2]
```diff
@@ -4,6 +4,29 @@ const prettyBytes = require('pretty-bytes')
 const BaseMicrobadgerService = require('./microbadger-base')
 const { NotFound } = require('..')
 
+const documentation = `
+<p>
+  The MicroBadger API can sometimes be a bit temparamental when it comes to retrieving the size of your image.
```
Comment (calebcartwright) [inline services/microbadger/microbadger-size.service.js]: I still think it's `..era..` if I'm not mistaken

Comment (PyvesB) [reply services/microbadger/microbadger-size.service.js]: Sorry, I'm struggling with this one tonight... 😕

Comment (calebcartwright) [reply services/microbadger/microbadger-size.service.js]: 😄 no worires

**Pull request merged**